test/cmdlineTests/combined_json_with_userdoc/c.sol

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.0;
+
+contract C {}
```

Comment:
Here again, I'd remove `c.sol` and put the contract in `input.sol`. Then add some natspec:
```suggestion
/// @notice Description for users.
contract C {}

/// @dev Description for developers.
contract D {}
```